src/main/app/forms/models/otherDamages.ts

```diff
@@ -1,14 +1,16 @@
 export class OtherDamages {
-  static readonly LESS = new OtherDamages('LESS', 'less')
-  static readonly MORE = new OtherDamages('MORE', 'more')
-  static readonly NONE = new OtherDamages('NONE', 'none')
+  static readonly LESS = new OtherDamages('LESS', 'less', 'thousandPoundsOrLess')
```

Comment:
the claim store has this as a different field, and as this is basically an enum it makes sense to store the information here